### ISSUE TITLE
International characters in file names

### ISSUE THREAD

**Issue by pvphome** [olivierkes/manuskript#1209]
I found that Manuscript saves files and directories with names that have international characters replaced with dashes. (This was tested under macOS). This makes it difficult to work with files using third-party programs. 
I suggest creating files and directories with names that use international characters.

**Comment by obw** [comment]
Not the best Idea, some network protocols have till today have Problems with this, also some versioning software programs.

There is a `folder.txt` file in every folder, where the real Name is placed...

**Comment by TheJackiMonster** [comment]
I agree that using these characters as replacement for names is intentional. It makes the directory structure safe to use in very different systems. So I wouldn't recommend changing that.

**Comment by TheJackiMonster** [comment]
I also don't understand how this naming of folders makes it difficult to work with the files in a third-party program. @pvphome Maybe you can elaborate?
